docs-ui/index.html

```diff
@@ -39,7 +39,7 @@
     window.onload = function() {
       // Begin Swagger UI call region
       const ui = SwaggerUIBundle({
-        url: "https://petstore.swagger.io/v2/swagger.json",
+        url: "/master/operator_api_swagger.json",
```

Comment:
Hey @achiurizo - this was propably changed in @mederic-p modifications for `v0.1` missed to port to master.

Note: we have gh-pages hosting this site https://developer.omisego.co/elixir-omg/ which shows all endpoints grouped by the app/mode.
It'd be great we push #690 soon-ish as we need to prepare `v0.2` specs...